tests/cython/atomspace/test_atomspace.py

```diff
@@ -358,31 +422,33 @@ def test_type(self):
     def test_strings(self):
         # set up a link and atoms
         tv = TruthValue(0.5, 0.8)
-        a1 = self.space.add_node(types.Node, "test1", tv)
+        a1 = Node("test1", tv)
 
-        a2 = self.space.add_node(types.Node, "test2")
+        a2 = Node("test2")
         a2.av = {"sti": 10, "lti": 1, "vlti": True}
         a2.tv = TruthValue(0.1, 0.3)
 
-        l = self.space.add_link(types.Link, [a1, a2])
+        l = Link(a1, a2)
+
+        last_uuid = 95
```

Comment:
this is the uuid of the atomspace, so I believe that you want `last_uuid = atomspace.get_uuid()` here.